conda_build/environ.py

```diff
@@ -118,7 +145,9 @@ def get_dict(m=None, prefix=None):
             d['CPU_COUNT'] = "1"
 
     if m.get_value('source/git_url'):
-        d.update(**get_git_build_info(d['SRC_DIR']))
+        d.update(**get_git_build_info(d['SRC_DIR'],
+                                      m.get_value('source/git_url'),
+                                      m.get_value('source/git_rev', default='master')))
```

Comment:
That was my first try, but it turns out [`git_rev` is a synonym for `git_tag`][1], and it's the canonical spelling.

[1]: https://github.com/stuarteberg/conda-build/blob/7ce7be2095848fc43b61da18492234a1694cef95/conda_build/metadata.py#L219-L228